classes/gateways/class.pmprogateway_check.php

```diff
@@ -108,9 +110,18 @@ static function pmpro_payment_option_fields($values, $gateway)
 			</th>
 			<td>
 				<textarea id="instructions" name="instructions" rows="3" cols="50" class="large-text"><?php echo wpautop(  wp_unslash( $values['instructions'] ) ); ?></textarea>
-				<p class="description"><?php esc_html_e('Who to write the check out to. Where to mail it. Shown on checkout, confirmation, and invoice pages.', 'paid-memberships-pro' );?></p>
+				<p class="description"><?php  esc_html_e(sprintf('Instructions for members to follow to complete their purchase, when paying with %s. Shown on checkout, confirmation, and invoice pages.', $check_gateway_label), 'paid-memberships-pro' );?></p>
```

Comment:
Only literal strings should be passed to i18n functions, not "calculated" strings (like those run through `sprintf()`).

```suggestion
				<p class="description"><?php  echo esc_html( sprintf( __( 'Instructions for members to follow to complete their purchase, when paying with %s. Shown on checkout, confirmation, and invoice pages.', 'paid-memberships-pro ), $check_gateway_label ) );?></p>
```